### DISCUSSION TITLE
Change to network share not working

### DISCUSSION THREAD

**Discussion by Operations8** [blakeblackshear/frigate#7604] [Ask A Question]
I am trying to change the location of my recordings and clips to my NAS.

So i did this:

https://docs.frigate.video/guides/ha_network_storage/

I looks like my frigate.db is used on my NAS, but Frigate keeps create a frigate/recordings directory in /media on my HASS machine.

The share is reachable and hass account is able to write.

Any ideas?

**Comment by NickM-27** [comment]
/media/Frigate is the expected location, it's just where the files from the NAS are mounted

**Reply by Operations8** [reply]
What do you mean? Is it not possible to change the location of the recording files? Not sure what you mean by "where the files from the NAS are mounted"

**Reply by NickM-27** [reply]
No you can't, because there's no reason to. /media/frigate is where frigate media is stored, and the docs guide exactly how to mount that location as the nas volume

**Comment by Operations8** [comment]
I did mount that as a NAS volume according to the docs guide. But it is not working.

**Reply by NickM-27** [reply]
In what way is it not working?

**Reply by Operations8** [reply]
Frigate keeps creating the recording folder in /media/frigate on the local HASS ssd.

**Reply by NickM-27** [reply]
How are you confirming it's on the SSD? And if that's the case, that's a supervisor issue not a frigate issue because frigate does not control the hosts file location

**Reply by Operations8** [reply]
Well i use SMB to check the local files on my HASS. So i see the recording folder reappear after restarting Frigate.

Plus i see no recording appearing on my NAS shared volume.

**Reply by NickM-27** [reply]
HA creates a virtual volume at the location so it would likely show up in SMB even if it's not being stored on the SSD

**Reply by Operations8** [reply]
Okee i wasn't clear enough :)

I can also tell by looking at the local SSD disc space. It is a 100% being stored on the local disc.  Because of my current issue with false positives frigate create 50GB+ per day. So it is easy to tell on a 120GB SSD.

**Comment by Operations8** [comment]
@NickM-27 , this trying to figure this out. When i look at the log it looks like frigate doesn't even try to find the mounted share.

```

2023-09-01 21:21:55.087278160  [2023-09-01 23:21:55] frigate.app                    INFO    : Starting Frigate (0.12.1-367d724)
2023-09-01 21:21:55.125708654  [2023-09-01 23:21:55] frigate.app                    INFO    : Creating directory: /media/frigate/recordings
2023-09-01 21:21:55.126754687  [2023-09-01 23:21:55] frigate.app                    INFO    : Creating directory: /media/frigate/clips
2023-09-01 21:21:55.128812010  [2023-09-01 23:21:55] frigate.app                    INFO    : Creating directory: /tmp/cache
2023-09-01 21:21:55.132848736  [2023-09-01 23:21:55] peewee_migrate                 INFO    : Starting migrations
2023-09-01 21:21:55.143160659  [2023-09-01 23:21:55] peewee_migrate                 INFO    : There is nothing to migrate
2023-09-01 21:21:55.172312501  [2023-09-01 23:21:55] detector.coral             

```

**Reply by NickM-27** [reply]
> When i look at the log it looks like frigate doesn't even try to find the mounted share

that's not how this works, Frigate doesn't do any of that itself. The job of the supervisor to mount the share. 

If the directions are followed correctly then the share is mounted at /media/frigate and frigate saves the media there

**Reply by Operations8** [reply]
I created the volume as "media" and called it frigate. That is what the doc said.
So i assume that is correct right?

**Reply by NickM-27** [reply]
yes, it would be expected to be storing on the NAS. If not then recommend checking supervisor logs. Users have already confirmed these steps to be working on the original request https://github.com/blakeblackshear/frigate/issues/6693

**Comment by Operations8** [comment]
There is nothing relevant in the supervisor log.
I did notice this:

```
23-09-01 22:44:48 WARNING (SyncWorker_0) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate (Full Access) Beta (0.12.0)
23-09-01 22:44:50 WARNING (SyncWorker_1) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate (Full Access) Beta (0.12.0)

```
Not sure if i am reading this right, but i didnt install the beta version.
